circle.yml

```diff
@@ -0,0 +1,25 @@
+dependencies:
+    cache_directories:
+       - "~/.opam"
+       - "~/aptcache"
+    override:
+        - sudo add-apt-repository -y ppa:avsm/ppa
+        - sudo mkdir -p ~/aptcache/partial
+        - sudo touch ~/aptcache/lock
```

Comment:
@yunxing I am slightly worried that this could be an issue in the future if the cache gets wiped. I might add them back in with a comment.